### PR TITLE
[REEF-2034] AdlsAccountName Expects FQDN

### PR DESCRIPTION
**Pull request by mandyshieh**
This addressed the issue by renaming the parameter AdlsAccountName to
AdlsAccountFQDN for clarity.

JIRA:
  [REEF-2034](https://issues.apache.org/jira/browse/REEF-2034)

Pull request:
  This closes #

### CODE REVIEW

File: lang/cs/Org.Apache.REEF.IO.Tests/TestAzureDataLakeFileSystem.cs [hunk 1]
```diff
@@ -210,13 +210,13 @@ public void TestGetFileStatusThrowsException()
 
         private sealed class TestContext
         {
-            public readonly string AdlAccountName = "adlAccount";
+            public readonly string AdlAccountFQDN = "adlAccount.azuredatalakestore.net";
```
Comment (tyclintw) [inline lang/cs/Org.Apache.REEF.IO.Tests/TestAzureDataLakeFileSystem.cs]: "AdlsAccountFqdn". You're missing the "s" in "adls". Also, the naming convention for acronyms is code is still camel case (see https://msdn.microsoft.com/en-us/library/ms229043(v=vs.100).aspx)

File: lang/cs/Org.Apache.REEF.IO/FileSystem/AzureDataLake/Parameters/DataLakeStorageAccountFQDN.cs [hunk 2]
```diff
@@ -23,7 +23,7 @@ namespace Org.Apache.REEF.IO.FileSystem.AzureDataLake.Parameters
     /// The account FQDN to be used to connect to the data lake store
     /// </summary>
     [NamedParameter("The account FQDN to be used to connect to the data lake store")]
-    internal sealed class DataLakeStorageAccountName : Name<string>
+    internal sealed class DataLakeStorageAccountFQDN : Name<string>
```
Comment (tyclintw) [inline lang/cs/Org.Apache.REEF.IO/FileSystem/AzureDataLake/Parameters/DataLakeStorageAccountFQDN.cs]: DataLakeStorageAccountFqdn

File: lang/cs/Org.Apache.REEF.IO.Tests/TestAzureDataLakeFileSystem.cs (hunk 1 above)
Comment (mandyshieh) [reply lang/cs/Org.Apache.REEF.IO.Tests/TestAzureDataLakeFileSystem.cs]: There is adls, adl, and adlfs scattered throughout, I'll change them all the adls to be consistent.

File: lang/cs/Org.Apache.REEF.IO/FileSystem/AzureDataLake/IDataLakeStoreClient.cs [hunk 3]
```diff
@@ -31,6 +31,6 @@ internal interface IDataLakeStoreClient
         /// <summary>
         /// Returns the account name for the AdlsClient.
```
Comment (singlis) [inline lang/cs/Org.Apache.REEF.IO/FileSystem/AzureDataLake/IDataLakeStoreClient.cs]: I would update the comment about the expected format.

Comment (mandyshieh) [reply lang/cs/Org.Apache.REEF.IO/FileSystem/AzureDataLake/IDataLakeStoreClient.cs]: Updated.

**Comment by homezcx** [comment]
@mandyshieh This PR needs rebase to resolve conflicts.

**Pull request merged**